client/chrome_trace_converter.cpp

```diff
@@ -12,14 +10,8 @@ namespace HawkTracer
 namespace client
 {
 
-ChromeTraceConverter::ChromeTraceConverter() :
-    _mapping_klass_name("HT_StringMappingEvent")
-{
-}
-
 ChromeTraceConverter::~ChromeTraceConverter()
 {
-    uninit();
```

Comment:
Don't we have to call stop in the destructor? We might have resource leak if user doesn't call stop()